.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,69 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+    # Skip jobs when only documentation files are changed
+    paths-ignore:
+      - '**.md'
+      - '**.rst'
+      - 'docs/**'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - '**.rst'
+      - 'docs/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        container: ["alicevision/popsift-deps:cuda10.2-ubuntu18.04", "alicevision/popsift-deps:cuda11.8.0-ubuntu20.04", "alicevision/popsift-deps:cuda12.1.0-ubuntu22.04"]
```

Comment:
I would recommend to put a date in the version of the popsift-deps image to keep versioning, like:

"alicevision/popsift-deps:cuda10.2-ubuntu18.04" => "alicevision/popsift-deps:2023.03.27-cuda10.2-ubuntu18.04"